easybuild/easyconfigs/s/Singularity/Singularity-3.5.3-GCC-8.3.0.eb

```
@@ -0,0 +1,59 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 		https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+# Software License::	Apache v2.0 and Custom
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Singularity'
+version = '3.5.3'
+
+github_account = 'sylabs'
+
+homepage = 'https://sylabs.io'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f057a15d28dae27b2e04cd1dd6a26e18daab41f89f16f07594302c2bcbb263f4']
+
+# single thread build to avoid segmentation during Golang package installation
+parallel = 1
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('libarchive', '3.4.2'),
+    ('Go', '1.14', '', True),
+]
+
+start_dir = '%(namelower)s-%(version)s'
+
+configure_cmd = "echo '%(version)s' > VERSION && "
+configure_cmd += "./mconfig -p '%(installdir)s' -b ./buildtree"
+
+prebuildopts = "cd ./buildtree && "
+
+preinstallopts = "cd '%(start_dir)s/buildtree' && "
```

@robqiao now the test fails because `start_dir` is not a template. you probably need to change it to this:
```suggestion
preinstallopts = "cd %(namelower)s-%(version)s/buildtree && "
```